tests_new/integration_tests/modules/s3_datasets/test_s3_tables_profiling.py

```diff
@@ -8,12 +8,36 @@
     get_table_profiling_run,
 )
 from integration_tests.errors import GqlError
-from integration_tests.modules.s3_datasets.conftest import DATASETS_TABLES_FIXTURES_PARAMS
+from integration_tests.utils import poller
 
 log = logging.getLogger(__name__)
 
 
-@pytest.mark.parametrize(*DATASETS_TABLES_FIXTURES_PARAMS)
+def has_job_finished(job):
+    return job.status not in ['STARTING', 'RUNNING', 'STOPPING', 'WAITING']
```

Comment:
Just curious, why do you check the statuses the job NOT IN rather then IN?

Reply:
I think it was because the "in-progress" where easy to write